### DISCUSSION TITLE
Add a TimePicker

### DISCUSSION THREAD

**Discussion by colinlienard** [chakra-ui/zag#1376] [Ideas]
It would be great to have a `TimePicker` component 

Example with the [MUI one](https://mui.com/x/react-date-pickers/time-picker):

<img width="472" alt="image" src="https://github.com/chakra-ui/zag/assets/64312634/e4144dc1-f657-4c9e-8d81-34151a017136">

I would be keen to help create this!

**Comment by pierresisson** [comment]
@segunadebayo @cschroeter hi guys, can we make a PR ?

**Reply by segunadebayo** [reply]
@colinlienard @pierresisson Yes please. Feel free to send in a PR for this component.

**Reply by pierresisson** [reply]
thank you! 🙏🏻

**Comment by segunadebayo** [comment]
Closing as duplicate of #2858.
